lib/shared/src/prompt/prompt-mixin.ts

```diff
@@ -5,7 +5,11 @@ import { PromptString, ps } from './prompt-string'
 /**
  * The preamble we add to the start of the last human open-end chat message that has context items.
  */
-const CONTEXT_PREAMBLE = ps`You have access to the provided codebase context. Question: `
+const CONTEXT_PREAMBLE = ps`You have access to my codebase context.`
```

Comment:
Can we keep this as before, because we tested that pretty thoroughly?
```suggestion
const CONTEXT_PREAMBLE = ps`You have access to the provided codebase context. `
```

Also I think we might be missing spaces between sentences.